app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadAdapter.kt

```diff
@@ -235,11 +235,11 @@ class ThreadAdapter(
         }
     }
 
-    private fun MessageViewHolder.bindMail(messageUi: MessageUi, position: Int) {
+    private fun MessageViewHolder.bindMail(messageUi: MessageUi, position: Int, isReducible: Boolean = false) {
```

Comment:
There's no point in adding a default value here.
```suggestion
    private fun MessageViewHolder.bindMail(messageUi: MessageUi, position: Int, isReducible: Boolean) {
```